cfsdk/src/main/java/io/harness/cfsdk/cloud/analytics/AnalyticsPublisherService.java

```diff
@@ -30,42 +29,25 @@ public class AnalyticsPublisherService {
 
     private static final Logger log = LoggerFactory.getLogger(AnalyticsPublisherService.class);
 
-    private static final String CLIENT;
-    private static final String SDK_TYPE;
-    private static final String SDK_VERSION;
-    private static final String SDK_LANGUAGE;
-    private static final String GLOBAL_TARGET;
-    private static final String TARGET_ATTRIBUTE;
-    private static final String FEATURE_IDENTIFIER_ATTRIBUTE;
-    private static final String FEATURE_NAME_ATTRIBUTE;
-    private static final String VARIATION_IDENTIFIER_ATTRIBUTE;
-    private static final String VARIATION_VALUE_ATTRIBUTE;
-
-    static {
-
-        CLIENT = "client";
-        SDK_TYPE = "SDK_TYPE";
-        TARGET_ATTRIBUTE = "target";
-        SDK_VERSION = "SDK_VERSION";
-        SDK_LANGUAGE = "SDK_LANGUAGE";
-        GLOBAL_TARGET = "__global__cf_target";
-        FEATURE_IDENTIFIER_ATTRIBUTE = "featureIdentifier";
-        FEATURE_NAME_ATTRIBUTE = "featureName";
-        VARIATION_IDENTIFIER_ATTRIBUTE = "variationIdentifier";
-        VARIATION_VALUE_ATTRIBUTE = "variationValue";
-    }
+    private static final String CLIENT = "client";
+    private static final String SDK_TYPE = "SDK_TYPE";
+    private static final String SDK_VERSION = "SDK_VERSION";
+    private static final String SDK_LANGUAGE = "SDK_LANGUAGE";
+    private static final String TARGET_ATTRIBUTE = "target";
+    private static final String FEATURE_IDENTIFIER_ATTRIBUTE = "featureIdentifier";
+    private static final String FEATURE_NAME_ATTRIBUTE = "featureName";
+    private static final String VARIATION_IDENTIFIER_ATTRIBUTE = "variationIdentifier";
+    private static final String VARIATION_VALUE_ATTRIBUTE = "variationValue";
```

Comment:
No big deal but `variationValue` isn't required in the evaluation metrics payload. https://harness.atlassian.net/wiki/spaces/FFM/pages/21070348431/Feature+Flag+SDKs#Metric-Attributes

Reply:
Might not be worth the effort to remove it. Up to you